### DISCUSSION TITLE
how can I display the barcode fig in a subplot?

### DISCUSSION THREAD

**Discussion by Lili-Cao** [chemosim-lab/ProLIF#201] [Q&A]
HI, 

I was trying to make subplots for barcode displlay, by passing a dict for subplots_kwargs, specifying a dict like this {"ax": axes[i // 3, i % 3]} 
however, the display methods does not recognize ax 

May I ask how does it work if I want to make subplots?

Best Regars
Lili

**Comment by cbouy** [comment]
The display_barcode function uses its own matplotlib figure and axis object so that won't work, but it also returns the axis object so you can probably move it to your own figure/subplots as shown [here](https://stackoverflow.com/a/46906599)

**Reply by Lili-Cao** [reply]
Cool. I will have a try, thank you!
